### PR TITLE
Add system/light themes

### LINKED ISSUE #32176: No way to activate bright mode when in dark mode
After https://github.com/nextcloud/server/pull/31751:

When your system in in dark mode, Nextcloud will be shown in dark mode as well because the default theming option respects this setting. There is unfortunatley no way to enable bright mode then anymore.

cc @skjnldsv @jancborchardt @nimishavijay how could we solve that?

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #32176 

image

### CODE REVIEW

File: apps/theming/lib/ThemingDefaults.php [hunk 1]
```diff
@@ -275,6 +275,16 @@ public function getLogo($useSvg = true): string {
...
 
+	/**
+	 * Default logo svg content
+	 *
+	 * @return string
+	 */
+	public function getDefaultLogoContent(): string {
+		$svg = file_get_contents(\OC::$SERVERROOT . $this->urlGenerator->imagePath('core', 'logo/logo.svg'));
+		return $svg ?: '';
+	}
```
Comment (skjnldsv) [inline apps/theming/lib/ThemingDefaults.php:286]: Unrelated, wrong push :see_no_evil: 
Waiting for design team's answer before pushing again

**Review by szaimen — APPROVED** [review]
I like :)
(Not tested yet though)

**Review by szaimen — APPROVED** [review]
lgtm and works :)

However in light mode are the icons in the settings app broken again?
image

**Comment by skjnldsv** [comment]
Failure unrelated

**Review by nickvergessen — APPROVED** [review]
Very nice, makes it work again for me

**Comment by nickvergessen** [comment]
> However in light mode are the icons in the settings app broken again?

again? They were broken all the time for me 😜 
But let's fix one thing after the other

**Comment by jancborchardt** [comment]
For a better overview, this split seems a bit nicer? (Also cause e.g. "System default light/dark with high contrast is not possible. That is currently if you change your system default theme and have a high contrast theme enabled, that is not switched automatically.)

---

**Theme**
- System default
- Light
- Dark

---

- [ ] Enable high contrast
- [ ] Enable dyslexia font

**Comment by skjnldsv** [comment]
> System default light/dark with high contrast is not possible

Yes it is with 25 :)

**Comment by szaimen** [comment]
I think what Jan was talking about is introducing a switch for high-contrast mode?

**Pull request merged**